src/utils/intl.js

```diff
@@ -0,0 +1,22 @@
+export let { ListFormat } = Intl;
+
+// Simple polyfill for Intl.ListFormat for en-US.
+//
+// Remove once more browsers support it:
```

Comment:
Arguably, the current browsers that support this are good enough?

Reply:
Sadly, Safari 14 doesn't have it yet (though 14.1 has it). We can wait a few months and this should no longer be a problem.